dedicated-examples/extended-error-tracking/README.MD

```diff
@@ -0,0 +1,55 @@
+# Extended Client-Side Error Tracking
+
+When a JavaScript error occurs on a tested web page, TestCafe stops test execution and posts an error message. Third-party code execution (e.g. controls, logging and analytics services) might cause unexpected errors that are not relevant to the tested app. You might want to filter those errors out to ignore them during the test.
```

Comment:
```suggestion
When a JavaScript error occurs on a tested web page, TestCafe stops test execution and posts an error message. Third-party code execution (e.g., controls, logging, and analytics services) may cause unexpected errors that are not relevant to the tested app. You might want to filter those errors out to ignore them during the test.
```

Reply:
You might want to filter those errors out to ignore them during the test.
This sentence doesn't make sense. Can you rewrite?

Reply:
> When a JavaScript error occurs on a tested web page, TestCafe stops test execution and posts an error message. Third-party code execution (for example, controls, logging, and analytics services) can cause unexpected errors that are not related to the tested app. You can filter these errors to ignore them during the test.

Reply:
You can filter these errors so that they are not included in the test.

Reply:
edited